src/plugins/chart_expressions/expression_metric/common/expression_functions/metric_vis_function.ts

```diff
@@ -37,6 +37,14 @@ export const metricVisFunction = (): MetricVisExpressionFunctionDefinition => ({
         defaultMessage: 'Which part of metric to color',
       }),
     },
+    colorFullBackground: {
+      types: ['boolean'],
+      default: false,
+      help: i18n.translate('expressionMetricVis.function.colorizeContainer.help', {
```

Comment:
Could you, please, change `expressionMetricVis.function.colorizeContainer.help` to `expressionMetricVis.function.colorFullBackground.help` for the consistency. 

The name after `function.*` is the name of the argument.

Reply:
done